Gemfile.lock

```diff
@@ -237,72 +238,72 @@ GEM
       timeout
     net-protocol (0.1.3)
       timeout
-    net-scp (3.0.0)
-      net-ssh (>= 2.6.5, < 7.0.0)
+    net-scp (4.0.0.rc1)
+      net-ssh (>= 2.6.5, < 8.0.0)
     net-smtp (0.3.1)
       digest
       net-protocol
       timeout
-    net-ssh (6.1.0)
+    net-ssh (7.0.1)
     nio4r (2.5.8)
-    nokogiri (1.13.3-x86_64-darwin)
+    nokogiri (1.13.8-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
     pagy (5.10.1)
       activesupport
     parallel (1.22.1)
-    parser (3.1.1.0)
+    parser (3.1.2.1)
       ast (~> 2.4.1)
-    pg (1.3.5)
+    pg (1.4.3)
     propshaft (0.6.4)
       actionpack (>= 7.0.0)
       activesupport (>= 7.0.0)
       rack
       railties (>= 7.0.0)
-    public_suffix (4.0.6)
-    puma (5.6.4)
+    public_suffix (5.0.0)
+    puma (5.6.5)
       nio4r (~> 2.0)
     racc (1.6.0)
-    rack (2.2.3)
-    rack-test (1.1.0)
-      rack (>= 1.0, < 3)
-    rails (7.0.2.3)
-      actioncable (= 7.0.2.3)
-      actionmailbox (= 7.0.2.3)
-      actionmailer (= 7.0.2.3)
-      actionpack (= 7.0.2.3)
-      actiontext (= 7.0.2.3)
-      actionview (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activestorage (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    rack (2.2.4)
+    rack-test (2.0.2)
+      rack (>= 1.3)
+    rails (7.0.4)
+      actioncable (= 7.0.4)
+      actionmailbox (= 7.0.4)
+      actionmailer (= 7.0.4)
+      actionpack (= 7.0.4)
+      actiontext (= 7.0.4)
+      actionview (= 7.0.4)
+      activejob (= 7.0.4)
+      activemodel (= 7.0.4)
+      activerecord (= 7.0.4)
+      activestorage (= 7.0.4)
+      activesupport (= 7.0.4)
       bundler (>= 1.15.0)
-      railties (= 7.0.2.3)
+      railties (= 7.0.4)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.4.2)
+    rails-html-sanitizer (1.4.3)
       loofah (~> 2.3)
-    railties (7.0.2.3)
-      actionpack (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    railties (7.0.4)
+      actionpack (= 7.0.4)
+      activesupport (= 7.0.4)
       method_source
       rake (>= 12.2)
       thor (~> 1.0)
       zeitwerk (~> 2.5)
     rainbow (3.1.1)
     rake (13.0.6)
-    ransack (3.0.0)
-      activerecord (>= 6.0.4)
-      activesupport (>= 6.0.4)
+    ransack (3.2.1)
+      activerecord (>= 6.1.5)
+      activesupport (>= 6.1.5)
       i18n
-    rb-fsevent (0.11.1)
+    rb-fsevent (0.11.2)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
-    redis (4.6.0)
-    regexp_parser (2.2.1)
+    redis (4.8.0)
+    regexp_parser (2.5.0)
```

Comment:
### [regexp_parser](https://github.com/ammar/regexp_parser)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.2.1 → 2.5.0

[_[change-log](https://github.com/ammar/regexp_parser/blob/master/CHANGELOG.md), [source-code](https://github.com/ammar/regexp_parser), [gem-diff](https://my.diffend.io/gems/regexp_parser/2.2.1/2.5.0)_]





<details>
<summary>Commits</summary>

A change of **67** commits. See the full changes on [the compare page](https://github.com/ammar/regexp_parser/compare/v2.2.1...v2.5.0).

These are the first 10 commits:
- (dceb026) [Refactor Syntax to skip instantiation](https://github.com/ammar/regexp_parser/commit/dceb02670d8f8f1b08660baa04ee67fd1306d298)
- (fa86d5c) [Remove lazy setting of extra version consts ...](https://github.com/ammar/regexp_parser/commit/fa86d5c2fb253620cc4521e27957411366dc6672)
- (b2d07a4) [Rename for clarity: Syntax.new -> Syntax.for](https://github.com/ammar/regexp_parser/commit/b2d07a42754d994645d14ec4d1dd99b18e1421da)
- (b950360) [Inherit features without class fiddling](https://github.com/ammar/regexp_parser/commit/b95036043ad4412d3db71a86d4622816cd2b24da)
- (6c67188) [Add some minimal benchmarks](https://github.com/ammar/regexp_parser/commit/6c6718812fbe79be894e248336e71db97f9c9c4f)
- (f3bdd78) [Extract ragel tasks to file](https://github.com/ammar/regexp_parser/commit/f3bdd78b471f6f4f42a470cd503d3ea3eab59593)
- (b2c16dc) [Move props task to file](https://github.com/ammar/regexp_parser/commit/b2c16dcf5cd0ecdfa986f6759c876060370d6937)
- (7b330fd) [Update CHANGELOG.md](https://github.com/ammar/regexp_parser/commit/7b330fd150bc8ced72981fdf0b56e0bb2ee7b300)
- (23ecbcf) [Add latest unicode properties](https://github.com/ammar/regexp_parser/commit/23ecbcf1be2d699a5320cf5c3ce90f31176bd384)
- (457dec6) [Fix deprecation warning](https://github.com/ammar/regexp_parser/commit/457dec6bb1832fddf1a7e8b6b1b967231f486496)


</details>